src/main/java/duke/Parser.java

```diff
@@ -15,96 +15,151 @@
 public class Parser {
 
     /**
-     * Executes the specified command and performs the corresponding action.
+     * Executes the 'list' command to display all tasks.
      *
-     * @param command The user command to be executed.
-     * @param tasks   The TaskList object to manage tasks.
+     * @param tasks The TaskList object containing tasks to list.
+     */
+    public static void executeList(TaskList tasks) {
```

Comment:
Well Done on using the proper naming convention for this function!

Reply:
Thank you!